OracleWebLogic/samples/12212-domain/container-scripts/domain.properties

```diff
@@ -0,0 +1,10 @@
+DOMAIN_NAME=abc_domain
+ADMIN_PORT=7001
+ADMIN_HOST=wlsadmin
+CLUSTER_NAME=DockerCluster
+PRODUCTION_MODE=dev
+ADMIN_PASSWORD=welcome4
```

Comment:
I actually have no problem with this: anyone who orchestrates can provide the same value to both containers. As long as it's clearly documented how to find the password and how to provide it to the managed server containers, it's all good with me. 

Which means I reckon you should just remove this line from `domain.properties` and just rely on documentation.